components/Layout.tsx

```diff
@@ -1,15 +1,35 @@
-import React from 'react'
+import React, {useEffect} from 'react'
 import Header from './Header/Header'
+import {useUser} from "@auth0/nextjs-auth0";
+import {userProfileState} from "../atoms/userProfile";
+import {useSetRecoilState} from "recoil";
+import {getUserFromFirebase} from "../lib/userHelper";
 
 interface LayoutProps {
     children: React.ReactNode
 }
 
-const Layout = ({ children }: LayoutProps) => {
+
+const Layout = ({children}: LayoutProps) => {
+    // useUser is used to get the current Auth0 user.
+    const {user, isLoading} = useUser();
+    // using useSetRecoilState to get userProfileState state from recoil.
+    const setUserProfileState = useSetRecoilState(userProfileState);
```

Comment:
I get the following error every time code imports an atom defined elsewhere:
```
Duplicate atom key "userProfileState". This is a FATAL ERROR in
      production. But it is safe to ignore this warning if it occurred because of
      hot module replacement.
```

Are you two getting it? Do we know why? Nothing we're doing diverges from the Recoil documentation I've seen...

Reply:
I see that too. I restart the app and it disappears, then it reappears again. Ill check online.
[Here](https://stackoverflow.com/questions/65506656/recoil-duplicate-atom-key-in-nextjs)